includes/class-wc-payments.php

```diff
@@ -1712,4 +1714,52 @@ public static function enqueue_assets_script() {
 			]
 		);
 	}
+
+	/**
+	 * Shows an alert notice for Hungarian, Sweden, and Czech Republic merchants on WooCommerce 7.4 and below
+	 */
+	public static function wcpay_show_old_woocommerce_for_hungary_sweden_and_czech_republic() {
+		$currencies        = [ 'HUF', 'SEK', 'CZK' ];
```

Comment:
I'm late again, but '$store_currency = strtolower( get_woocommerce_currency() );' would enable us to use currency constants and not hardcoded values here.

Reply:
Hmm, we are getting the store's currency but comparing it to a list of hard-coded values (the ones with a currency display problem -fixed in Core-). So, I don't think we can skip hard coding here 🤔

Reply:
Actually we can:

```php
		$currencies        = [ Currency::HUF, Currency::SEK, Currency::CZK ];
		$store_currency    = strtolower( get_woocommerce_currency() );
		$should_show_error = in_array( $store_currency, $currencies, true );

		if ( ! defined( 'WC_VERSION' ) || ! version_compare( WC_VERSION, '7.8', '<' ) || ! $should_show_error ) {
			return;
		}

		$notice = '';

		switch ( $store_currency ) {
			case Currency::HUF:
				/* translators: %1$s: The current WordPress version used by the store */
				$notice = __( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Hungarian business. Please update WooCommerce to version 7.8 or above (you are using %1$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
				break;
			case Currency::SEK:
				/* translators: %1$s: The current WordPress version used by the store */
				$notice = __( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Swedish business. Please update WooCommerce to version 7.8 or above (you are using %1$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
				break;
			case Currency::CZK:
				/* translators: %1$s: The current WordPress version used by the store */
				$notice = __( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Czech Republic business. Please update WooCommerce to version 7.8 or above (you are using %1$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
				break;
		}
```

Reply:
Aaah okay, I completely missed that when reading your comment. I'll draft a PR to fix this 👍

Reply:
Oh @kalessil , the `Currency` class isn't available on the client side :)

Reply:
I'm confused - PhpStrom did autocomplete with the constants from the server repository and after re-building indexes no more. My bad, sorry!